$OpenBSD: patch-sysdeps_openbsd_suid_open_c,v 1.4 2015/12/11 14:07:30 jasper Exp $

- Include missing headers for various defines and functions.
- Fix compilation because glibtop_machine definition has moved.

--- sysdeps/openbsd/suid_open.c.orig	Fri Dec 11 14:27:34 2015
+++ sysdeps/openbsd/suid_open.c	Fri Dec 11 14:30:49 2015
@@ -25,7 +25,10 @@
 #include <glibtop/cpu.h>
 #include <glibtop/open.h>
 #include <glibtop/init_hooks.h>
+#include <glibtop/machine.h>
 
+#include <fcntl.h>
+#include <kvm.h>
 
 /* !!! THIS FUNCTION RUNS SUID ROOT - CHANGE WITH CAUTION !!! */
 
